src/config/getWebpackClientConfig.js

```diff
@@ -10,7 +10,6 @@ const detectEnvironmentVariables = require("../lib/detectEnvironmentVariables");
 const getWebpackAdditions = require("../lib/getWebpackAdditions").default;
 const {
   additionalLoaders,
-  additionalPreLoaders,
```

Comment:
it so happens that none of our apps use pre-loaders, are they no longer a thing in webpack 2?